diagnostic.md

```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+Application router: store all the application routes
+Ember route: store the appropriate model
+
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+  ember g campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+  {{#link-to 'campuses'}}Back to Other Campuses {{/link-to}}
     ```
```

Comment:
`link-to` allows you to link to any defined route in your ember app. Here, we're asking to link to /campus/boston, so to get that you'd use `{{#link-to 'campus.boston'}}Boston{{/link-to}}`